src/Parser/Hunter/Marksmanship/Modules/Items/WarBeltOfTheSentinelArmy.js

```diff
@@ -5,8 +5,10 @@ import SPELLS from 'common/SPELLS';
 
 import Module from 'Parser/Core/Module';
 import Combatants from 'Parser/Core/Modules/Combatants';
-import { formatNumber } from 'common/format';
-import { formatPercentage } from "../../../../../common/format";
+import { formatNumber, formatPercentage } from 'common/format';
+import SpellLink from "../../../../../common/SpellLink";
```

Comment:
Ah, the IDE auto-imports 😅

Reply:
fuck... I need to configure that somewhere, or atleast make it not auto-hide imports so I catch these 😓